libs/ledger-live-common/src/families/celo/api/hubble.ts

```diff
@@ -50,20 +79,27 @@ const transactionToOperation = (
   const hasFailed = transaction.data.success
     ? !transaction.data.success
     : false;
-  const senders = transaction.data.from ? [transaction.data.from] : [];
-  const recipients = transaction.data.to ? [transaction.data.to] : [];
+  const data = transaction.data;
+  const sender = data?.Account || data?.from;
+  const recipient = data?.Group || data?.to;
+  //TODO: fetch/calculate fee from indexer when gas data is available
```

Comment:
this I need to do, right now I hardcoded fee for demo purposes but indexer started returning gas + gas price so I'll add calculation here